torch/utils/data/dataloader.py

```diff
@@ -601,9 +636,14 @@ def __init__(self, loader):
 
         assert self.num_workers > 0
 
+        if loader.multiprocessing_context is None:
+            multiprocessing_context = multiprocessing
```

Comment:
Can't we initialize the attribute with the `multiprocessing` module by default? You can make an exception for the `isinstance()` rule if it matches the module by identity.